examples/zone-printer/README.md

```diff
@@ -1,258 +1,250 @@
-[Creating and Managing Projects](https://cloud.google.com/resource-manager/docs/creating-managing-projects).
+2. **Set up `gcloud`**:
 
-After your project is created, run the following command in your
-terminal:
+    You can install `gcloud` by following the instructions at
+    [Installing Cloud SDK](https://cloud.google.com/sdk/downloads) and
+    initialize it by running the following commands:
 
-```shell
-PROJECT=<your-project-name>
-```
+    ```shell
+    gcloud init # Set $PROJECT as the default project
+    gcloud auth login
+    gcloud auth application-default login
+    ```
 
-### 0.2 `gcloud` installed and initialized
+3. Set up **`kubectl`**:
 
-You can install `gcloud` by following the instructions at
-[Installing Cloud SDK](https://cloud.google.com/sdk/downloads) and
-initialize it by running the following commands:
+    You can install `kubectl` by running:
 
-```shell
-gcloud init # Set $PROJECT as the default project
-gcloud auth login
-gcloud auth application-default login
-```
+    ```shell
+    gcloud components install kubectl
+    ```
 
-### 0.3 kubectl
+## 1. Create Kubernetes Clusters
 
-You can install `kubectl` by running:
+`kubemci` requires Kubernetes clusters that are v1.8.1 or newer. (You can check
+your cluster version using the `kubectl version` command.)
 
-```shell
-gcloud components install kubectl
-```
+You will need at least two Kubernetes clusters in two different compute [zones]
+to verify that `kubemci` works.
 
-### 0.4 Kubernetes clusters on GCP
+[zones]: https://cloud.google.com/compute/docs/regions-zones/
 
-#### Existing clusters
+#### Creating new GKE clusters
 
-> **Note:**: If you don't already have Kubernetes cluster on GCP,
-skip to the next subsection on [creating new clusters](#creating-new-clusters).
+The following commands will create two clusters:
 
-`kubemci` requires Kubernetes clusters that are v1.8.1 or newer. Please
-ensure that all the clusters you are using in this tutorial satisfy the
-minimum version requirement. You can check your cluster version using
-the `kubectl version` command.
+- `cluster-1` in `us-east4-a`
+- `cluster-2` in `europe-west1-c`:
 
-If you already have clusters with their credentials in your local
-kubeconfig (`$HOME/.kube/config` by default) file, you need to ensure
-that you either only have clusters you are going to use for this
-tutorial in your default kubeconfig file or create a new kubeconfig
-file that just contains credentials for those clusters. Assuming that
-the clusters you are going to use for this tutorial are called
-`us-east`, `eu-west` and `asia-east`.
+```shell
+# Create a cluster in us-east and get its credentials
+gcloud container clusters create \
+    --cluster-version=1.9 \
+    --zone=us-east4-a \
+    cluster-1
 
-List the contexts available (note the names of the ones you wish to
-use with kubemci)
+KUBECONFIG=clusters.yaml gcloud container clusters get-credentials \
+    --zone=us-east4-a \
+    cluster-1
```

Comment:
I am wondering if these two commands could just be collapsed into:

```shell
KUBECONFIG=clusters.yaml gcloud container clusters create \
   --cluster-version=1.9 \
   --zone=us-east4-a \
   cluster-1
```

Does that work? Simplifies even further.

Thanks for this reorganization btw.

Reply:
looks like it can be.

Reply:
Cool!